docs/rst/notes/iron/notes.rst

```diff
@@ -1,6 +1,6 @@
 .. _notes_iron_latest:
```

Comment:
I am missing here the `https://github.com/eProsima/rosidl.git` new package

Reply:
Added in 6d3682d